src/store/reducers/modalReducer.js

```diff
@@ -38,6 +38,13 @@ function modalReducer(state = initialState, action) {
       return Object.assign({}, state, intiateResetPayload);
     case (PASSWORD_RESET):
       return Object.assign({}, state, passwordResetPayload);
+    case (EDIT_COMMENT):
+      return ({
+        ...state,
+        editData: action.payload,
+        component: 'edit-comment',
+        modalShow: true,
+      });
     default:
```

Comment:
By default I believe you should just return the current state without changing anything. You are returning nothing, and this might be a problem.